src/status_im/contexts/profile/config.cljs

```diff
@@ -40,9 +40,6 @@
             :verifyENSURL             config/verify-ens-url
             :verifyENSContractAddress config/verify-ens-contract-address
             :verifyTransactionChainID config/verify-transaction-chain-id
-            :upstreamConfig           config/default-network-rpc-url
-            :networkId                config/default-network-id
```

Comment:
I see a few usages of `networkId` in `status-go`. But, removing this doesn't affect it as it sets the default value (Chain ID `1`).